src/overlays/actors/ovl_En_Daiku2/z_en_daiku2.h

```diff
@@ -5,9 +5,40 @@
 
 struct EnDaiku2;
 
+typedef void (*EnDaiku2ActionFunc)(struct EnDaiku2*, GlobalContext*);
+
+typedef struct EnDaiku2Particle {
+    /* 0x00 */ u8 enabled;
+    /* 0x04 */ Vec3f unk_04;
+    /* 0x10 */ Vec3f unk_10;
+    /* 0x1C */ Vec3f unk_1C;
+    /* 0x28 */ Vec3s unk_28;
+    /* 0x30 */ f32 unk_30;
+    /* 0x34 */ s16 unk_34;
+} EnDaiku2Particle; // size = 0x38
+
 typedef struct EnDaiku2 {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0xC84];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[17];
+    /* 0x1EE */ Vec3s morphTable[17];
+    /* 0x254 */ EnDaiku2ActionFunc actionFunc;
+    /* 0x258 */ Path* unk_258;
+    /* 0x25C */ s16 unk_25C;
+    /* 0x260 */ f32 unk_260;
+    /* 0x264 */ f32 unk_264;
+    /* 0x268 */ Vec3f unk_268;
+    /* 0x274 */ s16 unk_274;
+    /* 0x276 */ s16 unk_276;
+    /* 0x278 */ s16 unk_278;
+    /* 0x27A */ s16 unk_27A;
+    /* 0x27C */ char unk_27C[0x4];
```

Comment:
```suggestion
    /* 0x27C */ UNK_TYPE1 unk_27C[0x4];
```
I believe we have been deviating away from leaving them as char and putting them as `UNK_TYPE1`